### ISSUE TITLE
Recipe View Rework

### ISSUE THREAD

**Issue by vabene1111** [TandoorRecipes/recipes#2220]
- [ ] some kind of compact mode #2175
- [ ] #1955

**Comment by tomtjes** [comment]
Not sure if this project is still relevant with Tandoor 2.0, but I wanted to bump it for #1955. Until recently, it was possible to achieve it with custom CSS, hiding the bubbles and making the print view notes visible. That's no longer an option, so a toggle would be appreciated. 

I'd advocate for it being a space or user setting rather than a toggle in the recipe view, because I think it depends on how someone decides to use the notes fields on their instance, i.e. do they typically only contain _good to know_ information, or are they mostly prep instructions that you frequently rely on while cooking.

**Comment by vabene1111** [comment]
makes sense. I tought about adding it to the steps overview when working on that or do you think it should be visible everywhere?

**Comment by tomtjes** [comment]
Do you mean, the option to show the notes instead of the bubbles should only affect the steps overview? I don't think so. It would be nice to have it affect the steps too.
If you mean, should the toggle be in the steps overview, that would probably make sense, if at the same time, a default can be set per space.
(Sorry, it took me so long to get back to you. Also on other things I wanted to get my hands on. Life happened.)

**Comment by vabene1111** [comment]
does for me all the time as well, no worries. I know this is a hot topic, will look into it again.
